python/tutorials/09-illegal-memory-access-analysis.py

```diff
@@ -0,0 +1,84 @@
+"""
+Invalid Memory Access Analysis
+===============
+
+In this tutorial, you will learn utlities to detect invalid memory accesses, including:
+1. `triton.enable_invalid_memory_access_analysis` that catches exceptions on-the-fly and print the problematic line number.
```

Comment:
This is implied, but might want to state that not all invalid memory accesses produce exceptions, so this won't catch all of them.

Reply:
Yeah, makes sense